RetailCoder.VBE/UI/RubberduckUI.resx

```diff
@@ -2071,7 +2071,7 @@ Would you like to import them to Rubberduck?</value>
   <data name="CodeExplorer_ExportAll" xml:space="preserve">
     <value>Export Project...</value>
   </data>
-  <data name="HotkeyDescription_ExportActiveProject" xml:space="preserve">
+  <data name="HotkeyDescription_ExportAllCommand" xml:space="preserve">
```

Comment:
Ideally, I'd like to see these hotkey-specific i18n definitions die, because they should become obsolete